features/built_in_matchers/class.feature

```diff
@@ -0,0 +1,56 @@
+Feature: Class matchers
+
+  rspec-expectations a matcher to specify the nature of classes:
+
+    * `expect(Klass).to be_subclass_of(Klazz)`: calls `Klass.ancestors.include?(Klazz)`, which returns true if
+        Klazz is in Klass's class hierarchy or is a module and is included in a class in Klass's
+        class hierarchy. This matcher is aliased as `be_a_subclass_of` and `a_subclass_of`.
+
+
+  Scenario: be_(a_)subclass_of matcher
+    Given a file named "be_subclass_of_matcher_spec.rb" with:
+      """ruby
+      module Qux
+
+      end
+
+      class Foo
+
+      end
+      
+      class Bar < Foo
+
+      end
+
+      class Baz < Bar
+        include Qux
+      end
```

Comment:
These module and class definitions are taking up a lot of room.  I'd prefer to see them in a more compact style like `module Qux; end`.